daemon/cmd/daemon.go

```diff
@@ -369,6 +368,10 @@ func NewDaemon(ctx context.Context, epMgr *endpointmanager.EndpointManager, dp d
 	// created.
 	isKubeProxyReplacementStrict := initKubeProxyReplacementOptions()
 
+	// LB map init must come right after kube-proxy replacement initialization
+	// given the default LB map size could still change there.
```

Comment:
:thinking: I didn't see the corresponding diff for this comment. Is this true? Is it a potential bug in lbmap initialization on master? (or did I just miss the corresponding diff?)

Reply:
This is another leftover from the initial implementation of Maglev. Previously, in 0c37977dd860e750f12eb30806b40750e06387b0 the lbmap size could have been configured in `initKubeProxyReplacement(..)`. Removing.